js/src/wp-seo-post-scraper.js

```diff
@@ -24,6 +24,10 @@ import isContentAnalysisActive from "./analysis/isContentAnalysisActive";
 import snippetPreviewHelpers from "./analysis/snippetPreview";
 import UsedKeywords from "./analysis/usedKeywords";
 import { setMarkerStatus } from "./redux/actions/markerButtons";
+import {
```

Comment:
Is the line too long when this is import is on one single line? It seems like an overkill when a single import from a file is split into 3 lines.

Reply:
PHPStorm does this automatically :(